contracts/Response.sol

```diff
@@ -154,12 +181,18 @@ contract Response {
   function createQuestion(
     uint64 twitterUserId, bytes32 content, address foundation, uint amount
   ) payable external {
+    // read uint backend fee and make sure it is same as msg.value
     require(msg.value == s.rUint(BACKEND_FEE));
+    // transfer from sender to current address token amount
     require(token.transferFrom(msg.sender, this, amount));
+    // read boolean of foundation addressto check whether it is registered
     require(s.rBool(FOUNDATION, foundation));
     // todo twitter user id and content is unchecked
+    // QUESTION: what will it be checked for, same as foundation boolean?
```

Comment:
We cannot anyhow validate question content because now it stored in a bytes32 variable instead of a string. The same with userId. It is more a remark for a front-end developer or proposal to implement some moderation interface.